src/pages/Create.js

```diff
@@ -0,0 +1,424 @@
+import FormatItalicIcon from '@material-ui/icons/FormatItalic';
+import FormatUnderlinedIcon from '@material-ui/icons/FormatUnderlined';
+import TextFields from '@material-ui/icons/TextFields';
+import InsertLink from '@material-ui/icons/InsertLink';
+import Close from '@material-ui/icons/Close';
+import CreateMenu from "../components/CreateMenu";
+import { createEditor, Editor, Transforms, Text, Range } from "slate";
+import {Slate, Editable, withReact, useSlate, ReactEditor} from "slate-react";
+import { useTheme } from '@material-ui/core/styles';
+
+
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        '& .MuiTextField-root': {
+            margin: theme.spacing(1),
+            width: '25ch',
+        },
+    },
+    titleField: {
+        fontSize: 40,
+        outline: 'none',
+        border:'none',
+
+    },
+    subtitleField: {
+        fontSize: 30,
+        outline: 'none',
+        border:'none',
+    },
+    bodyField: {
+        fontSize: 20,
+        outline: 'none',
+        border:'none',
+    },
+    formatMenu: {
+        position: 'absolute',
+        opacity: 0,
+        zIndex: 1,
+        top: -10000,
+        left: -10000
+    }
+}));
+
+export default function Create (props) {
+    const classes = useStyles();
+
+    //states to track store the blog text
+    const [title, setTitle] = useState();
+    const [subtitle, setSubtitle] = useState();
+    const [blogBody, setBlogBody] = useState([{
+        type: 'paragraph',
+        children: [{ text: '' }],
+    },]);
+
+    const editorRef = useRef();
+    if (!editorRef.current) editorRef.current = withReact(createEditor());
+    const editor = editorRef.current;
+
+    const handleTitleChange = (event) => {
+        setTitle(event.target.value);
+    }
+    const handleSubtitleChange = (event) => {
+        setSubtitle(event.target.value);
+    }
+
+    const renderElement = useCallback(props => {
+        switch (props.element.type) {
+            case 'code':
+                return <CodeElement {...props} />
+            case 'title':
+                return <TitleElement {...props} />
+            case 'subtitle':
+                return <SubtitleElement {...props} />
+            default:
+                return <DefaultElement {...props} />
+        }
+    }, []);
+
+    const renderLeaf = useCallback(props => {
+        return <Leaf {...props} />
+    }, []);
+
+    return (
+        <Container maxWidth={'sm'} style={{marginTop:14}}>
+            <Grid container direction={'row'} alignItems={'center'} >
+                <TextField label={'Title'} fullWidth={true} autoFocus={true} multiline={true} rowsMax={3}
+                           onChange={handleTitleChange}
+                           InputProps={{style: {fontSize: 40,marginBottom: 12}}}
+                />
+            </Grid>
+            <Grid container direction={'row'} alignItems={'center'}>
+                <TextField label={'Subtitle'} fullWidth={true} multiline={true} rowsMax={4}
```

Comment:
Same, the components of this page could probably be refactored into common components. Then `Create.js` becomes much more light weight and when we add the ability to edit a blog we can simply pull in the common components. We also then ensure the same UX when creating vs editing a blog.